src/applications/disability-benefits/2346/containers/OrderHistory.jsx

```diff
@@ -0,0 +1,228 @@
+import ErrorableCheckbox from '@department-of-veterans-affairs/formation-react/ErrorableCheckbox';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { focusElement } from 'platform/utilities/ui';
+import {
+  getDLCAccessoriesData,
+  getDLCBatteryData,
+  // NOTE: Decision was to take socks out of MVP -@maharielrosario at 1/28/2020, 9:49:20 AM
+  // getDLCSocksData,
+  updateDLCData,
+} from '../actions';
+
+class OrderHistory extends Component {
+  // NOTE: Using React state for checkboxes -@maharielrosario at 1/8/2020, 7:47:21 PM
+  // When discussed with other devs, the concensus was to have a stateful
+  // component to house our checkboxes' state. But all other state is in Redux.
+  // Is there an easy way to house individual checkbox state in Redux?
+
+  componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
+    this.props.getDLCBatteryData();
+    // NOTE: Decision was to take socks out of MVP -@maharielrosario at 1/28/2020, 9:49:20 AM
+    // this.props.getDLCSocksData();
+    this.props.getDLCAccessoriesData();
+  }
+
+  createDLCUIData = dlcData => {
+    const dlcUIData = dlcData.map(dlcDataObj => {
+      let checkbox;
```

Comment:
Does this actually work with checkbox being within the scope of ```map``` ?  Should checkbox be outside of the map block?

Reply:
This file has been removed.